### PR TITLE
sync the jupyterlab version to 3.0.16

### PR DESCRIPTION
**Pull request by harshad16**
sync the jupyterlab version to 3.0.16
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

This is to keep the version pinned down.
This would be tested in the notebook.
We need to release the tag after we merge this.

- [x] The commits are squashed in a cohesive manner and have meaningful messages.
- [x] JIRA link(s): https://issues.redhat.com/browse/RHODS-1607
- [x] The Jira story is acked
- [x] An entry has been added to the latest build document in [Build Announcements Folder](https://drive.google.com/drive/folders/1sgkK1WZgGo9CXsLizNe0GbAzVKuSKrZL).
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious)

### CODE REVIEW

**Comment by harshad16** [comment]
we can ignore the build-check, it is not configured properly.

**Comment by tmckayus** [comment]
The only issue is to make sure that this changes none of the versions noted in the imagestream annotations for the pytorch notebook image (used by tooltips)

**Comment by tmckayus** [comment]
Here is the updated imagestream annotation from odh-deployer.

{"name":"PyTorch","version":"1.8.1"},{"name":"Tensorboard","version":"1.15.0"},{"name":"Boto3","version":"1.17.11"},{"name":"Kafka-Python","version":"2.0.2"},{"name":"Matplotlib","version":"3.4.1"},{"name":"Numpy","version":"1.20.2"},{"name":"Pandas","version":"1.2.3"},{"name":"Scikit-learn","version":"0.24.1"},{"name":"Scipy","version":"1.6.2"}

numpy in Pipfile.lock is 1.19.2
pandas in Pipfile.lock is 1.2.4

File: Pipfile [hunk 1]
```diff
@@ -37,9 +38,9 @@ scipy = "==1.6.2"
 seaborn = "*"
 sqlalchemy = "*"
 statsmodels = "*"
-tensorboard = "<2.0"
-torch  = "*"
-torchvision= "*"
+tensorboard = "==1.15.0"
+torch  = "==1.8.1"
```
Comment (tmckayus) [inline Pipfile:42]: can we pin the torch version to 1.9.0 to preserve the version currently listed in the imagestream annotations? (is 1.8.1 essential for some reason)?

**Comment by tmckayus** [comment]
Looks like all the version match, except for the torch version itself which is currently 1.9.0

File: Pipfile (hunk 1 above)
Comment (harshad16) [reply Pipfile:42]: torch 1.8.1 is required for compatibility with cuda 11.0.3 
i know we are not having gpu in GA , but having a consistency might be good.

Another thing, where did you find torch to be 1.9.0 , check the imagestream annotation in odh-deployer.
https://github.com/red-hat-data-services/odh-deployer/blob/1d6007f3fea4da6c7935075407ebf6be20f74505/jupyterhub/cuda-11.0.3/manifests.yaml#L129

Comment (tmckayus) [reply Pipfile:42]: ah I read it from odh-manifests, my mistake

Comment (tmckayus) [reply Pipfile:42]: ah, I looked in in odh-manifests.

Corrected above with the list from odh-deployer, we still have a discrepancy for numpy and pandas.  We are trying to keep versions consistent across notebooks and consistent with the tooltips, so we need to know why those are different and if they are necessary.

Doesn't look like they were changed in this PR so we need to go back and find where they changed.

Comment (harshad16) [reply Pipfile:42]: if you view this pr : https://github.com/red-hat-data-services/odh-deployer/pull/171
it fixes the tooltip as well, i have made sure on the version which we had decided earlier

**Review by tmckayus — APPROVED** [review]

**Pull request merged**